tests/network/expose.go

```diff
@@ -520,12 +522,13 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				for _, node := range nodes.Items {
 					Expect(node.Status.Addresses).ToNot(BeEmpty())
 					nodeIP := node.Status.Addresses[0].Address
-
+					launcher, err := libvmi.GetPodByVirtualMachineInstance(udpVM, udpVM.GetNamespace())
```

Comment:
Why this is was extracted out of the IPv6 condition?

Reply:
Done